support/client/lib/vwf/model/kineticjs.js

```diff
@@ -2162,7 +2174,13 @@ define( [ "module",
                 }
             }
 
-            // Redraw the object now that it's image has loaded
+            // Redraw the object now that its image has loaded
+            if ( kineticObj.attrs.hitGraphFromCache && kineticObj.isVisible() ) {
+                kineticObj.clearCache();
+                kineticObj.draw();
+                kineticObj.cache();
+                kineticObj.drawHitFromCache();
+            }
             kineticObj.draw();
```

Comment:
Can this be in an `else` condition to avoid drawing twice?

Reply:
The first draw is just to render the drawing for the cache, the second draw is to ensure that the hitGraph has been rendered from the cached image at least once.